spikeinterface/preprocessing/normalize_scale.py

```diff
@@ -204,8 +206,18 @@ class ZScoreRecording(BasePreprocessor):
         The recording extractor to be centered
     mode: str
         "median+mad" (default) or "mean+std"
-    dtype: str or np.dtype
-        The dtype of the output traces. Default "float32"
+    dtype: None or dtype
+        If None the the parent dtype is kept.
+        For integer dtype a int_scale must be also given.
+    gain : None or np.array
+        Pre-computed gain.
+    offset : None or np.array
+        Pre-computed offset
+    int_scale : None or float
+        Apply a a final scale to fit integer range.
+        This is usefull for dtype=int and want to keep the output as int also typical scale are 200 which means
+        that the noise in noise [0-1] will be encoded in [0-200].
```

Comment:
```suggestion
    int_scale : None or float
        Apply a scaling factor to fit the integer range.
        This is used when the dtype is an integer, so that the ourput is scaled. 
        For example, a value of `int_scale=200` will scale the zscore value to a standard deviation od 200.
```